app/src/main/java/com/example/drcreditdev/creditCardUI/creditPage/Credit_home_page.kt

```diff
@@ -66,15 +65,15 @@ class credit_home_page : AppCompatActivity() {
             &&intent.getStringExtra("year")!=null)
         {
 
-          date.text = intent.getStringExtra("day").toString()+intent.getStringExtra("month")+"'"+intent.getStringExtra("year")
+          date.text = intent.getStringExtra("day").toString()+" "+intent.getStringExtra("month")+"'"+intent.getStringExtra("year")
         }
         else
         {
-            date.text=sharedpreferences.getString("day","12").toString()+sharedpreferences.getString("month","Apr").toString()+"'"+sharedpreferences.getString("year","22").toString()
```

Comment:
shift all business logic to Viewmodel, the responsibility of view should be to show ciew only, no logic calculations.

Reply:
In pending but working on it!